MDANSE/Tests/UnitTests/TrajectoryEditor/test_editor.py

```diff
@@ -98,9 +100,14 @@ def test_editor_frames():
     original.close()
     changed.close()
 
-    with h5py.File(temp_name + ".mdt") as actual,  h5py.File(short_traj) as desired:
-        np.testing.assert_array_almost_equal(actual["/configuration/coordinates"], desired["/configuration/coordinates"][0:501:10])
-        np.testing.assert_array_almost_equal(actual["/time"], desired["/time"][0:501:10])
+    with h5py.File(temp_name + ".mdt") as actual, h5py.File(short_traj) as desired:
```

Comment:
Thinking we probably want a test-global `compare_mdt` function we can use here and elsewhere. Not this PR.

Reply:
Yea I think I agree with this.